src/controllers/orderController.js

```diff
@@ -57,11 +57,68 @@ const createOrder = async (req, res) => {
   res
     .status(StatusCodes.CREATED)
     .json({ clientSecret: paymentIntent.client_secret, order });
+  /*
+#swagger.summary = 'Create a new order and process payment'
+#swagger.description = 'Creates a new order and processes payment using Stripe.'
+#swagger.tags = ['Orders']
+#swagger.parameters['body'] = {
```

Comment:
For swagger-autogen comments, we don't use the full Swagger/OpenAPI syntax to define objects. Rather, the library wants us to just describe the actual shape of the object with example data, and it will infer the type and all that.

Reply:
Akos , I have three problems in swagger-output.json in lines 531, 539, 565. As for me , it seems the same like other schemas. Could you please check? thank you